web/scalica/micro/models.py

```diff
@@ -58,9 +60,44 @@ def decrement_num_upvotes(sender, instance, created, **kwargs):
     """
     instance.answer.num_upvotes -= 1
 
-class School(models.Model):
+class University(models.Model):
     name = models.CharField(max_length=255)
 
+class MyUserCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        help_texts = {
+            'username' : '',
+        }
+
+"""
+MyUserCreationForm and ProfileForm can be used in the same view for user creation
+Reference: https://simpleisbetterthancomplex.com/tutorial/2016/07/22/how-to-extend-django-user-model.html
+"""
+class ProfileForm(ModelForm):
+    class Meta():
+	model = Profile
+	fields = ['first_name', 'last_name', 'email', 'university']
+
+class QuestionForm(ModelForm):
+    class Meta():
+        model = Question
+	fields = ['text']
+	widgets = {
```

Comment:
some spacing weirdness between line 83 & 84

Reply:
this is what happens when I try to edit in Vim